src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -4,55 +4,34 @@ import { useUID } from 'react-uid';
 import styles from './FiltersDrawer.module.css';
 import Button from '../Button';
 import ButtonGroup from '../ButtonGroup';
-import Checkbox from '../Checkbox';
 import Drawer from '../Drawer';
-import FiltersCheckboxField from '../FiltersCheckboxField';
 import Heading from '../Heading';
 
 export type Props = {
   /**
-   * Checkbox fields that will be displayed in the filters drawer.
+   * Checkboxes or Checkbox fields that will be displayed in the filters drawer.
```

Comment:
Will these always be checkboxes or can they be radios, toggles, etc?

Reply:
Think they could be any of those you mentioned, will update description